docs/modules/api_gateway.md

```diff
@@ -89,20 +90,25 @@ The following arguments are supported:
 * `routes` - A mapping of path prefixes to `route` mappings to define the behavior at that path prefix (The leading '/' is cosmetic. Paths can contain up to five levels deep.).
 * `tags` - (optional) A mapping of tags to be applied to all resources.
 * `log_retention_in_days` - (optional, default 0) - number of days to retain logs. 0 (the default) means to never expire logs. Other valid values are: 1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653.
+* `authorizers` - A mapping of string keys to an attribute mapping. The keys are arbitrary for reference in `route` entries.
 * `lambdas` - A mapping of string keys to an attribute mapping. The keys are arbitrary for reference in `route` entries.
 * `set_cloudwatch_role` - A boolean indicating if the API Gateway Cloudwatch role should be set
 * `apigateway_cloudwatch_role_arn` - (optional) If `set_cloudwatch_role` is true, then specifying this will set the specific role. If not provided, a role will be created.
 * `redeployment_hash` - (optional) - Entropy variable to trigger a deployment to be made. If omitted, this module will do its best to detect applicable changes.
 
-The `lambda` attribute map contains:
+The `authorizers` attribute map contains:
```

Comment:
What's the reason for breaking this into a separate input? Since you pointed out that it's a breaking change I'm guessing there's a reason.